onnxscript/onnx_opset.py

```diff
@@ -23,24 +23,16 @@
~opset9 = values.Opset("", 9)
~opset10 = values.Opset("", 10)
~opset11 = values.Opset("", 11)
 opset12 = values.Opset("", 12)
 opset13 = values.Opset("", 13)
 opset14 = values.Opset("", 14)
-
-if onnx_opset_version() >= 15:
-    opset15 = values.Opset("", 15)
-
-if onnx_opset_version() >= 16:
-    opset16 = values.Opset("", 16)
-
-if onnx_opset_version() >= 17:
-    opset17 = values.Opset("", 17)
-
-if onnx_opset_version() >= 18:
-    opset18 = values.Opset("", 18)
+opset15 = values.Opset("", 15)
```

Comment:
Not sure I understand this. Won't the parent package/module be imported before a child package is imported?

Reply:
It looks ok to me

Reply:
I think it should be fine

Reply:
Yes, I think it is fine. I think this may be case that CodeQL could handle better if it distinguishes child-parent dependency from other import dependency?

Reply:
But how do we turn off this error message? The CI fails because of this.

Reply:
lgtm is non-blocking. they won't show next time. We can also click the "dismiss alert" button